lib/Installation/Partitioner/LibstorageNG/SuggestedPartitioningPage.pm

```diff
@@ -28,13 +28,37 @@ sub press_guided_setup_button {
     send_key('alt-g');
 }
 
+=head2 select_start_with_existing_partitions
+
+  select_start_with_existing_partitions()
+
+Opens existing partition of the Expert Partiotioner from the Suggested Partitioning page .
+
+=cut
+
 sub select_start_with_existing_partitions {
     my ($self) = @_;
     assert_screen($self->SUGGESTED_PARTITIONING_PAGE);
     send_key 'alt-e';
     send_key 'p';
 }
 
+
+=head2 select_start_with_curremt_partitions
+
+  select_start_with_existing_partitions()
+
+Opens current suggestion of the Expert Partiotioner from the Suggested Partitioning page .
```

Comment:
It's actually current partitions, not suggestion. Don't you mind to improve this description?

Reply:
i tried to use the term of the UI. one is existing_partition and the other is current_suggestion. change done